backend/requirements.txt

```diff
@@ -1,36 +1,3 @@
-alembic==1.13.2
-annotated-types==0.7.0
-anyio==4.4.0
-click==8.1.7
-fastapi==0.112.2
-h11==0.14.0
-idna==3.8
-Mako==1.3.5
-MarkupSafe==2.1.5
-psycopg2-binary==2.9.9
-pydantic==2.8.2
-pydantic_core==2.20.1
-sniffio==1.3.1
-SQLAlchemy==2.0.32
-starlette==0.38.2
-typing_extensions==4.12.2
-uvicorn==0.30.6
-dnspython==2.6.1
-email_validator==2.2.0
-fastapi-cli==0.0.5
-httpcore==1.0.5
-httptools==0.6.1
-httpx==0.27.0
-Jinja2==3.1.4
-markdown-it-py==3.0.0
-mdurl==0.1.2
-Pygments==2.18.0
-python-dotenv==1.0.1
-python-multipart==0.0.9
-PyYAML==6.0.2
-rich==13.8.0
-shellingham==1.5.4
-typer==0.12.5
-uvloop==0.20.0
-watchfiles==0.23.0
-websockets==13.0
+fastapi==0.114.1
```

Comment:
I think we should use fastapi[standart] instead of only fastapi. It comes with some usefull development tools, like the fastapi-cli. What do you think?